main.go

```diff
@@ -136,6 +139,11 @@ func main() {
 		setupLog.Error(err, "Unable to create controller", "controller", "Service")
 		os.Exit(1)
 	}
+
+	if enablePodReadinessWebhook {
```

Comment:
i think it's better to move such configuration enablePodReadinessWebhook  inside inject itself.(similar style as our appmesh's inject configurations like sidecar image).

also, the injector's setupWithManager should always run no matter it's enabled or not. since it setups the webhook servers, if it not running and our webhook's manifests is installed, pod creation requests will be blocked

Reply:
sounds fair. I will move it to inject.